src/_data/platforms.yml

```diff
@@ -1,50 +1,50 @@
 - platform: 'Android SDK'
   chipsets: 'x64, Arm32, Arm64'
   supported: '21 to 34'
-  besteffort: 'N/A'
+  ci-tested: '21 to 34'
   unsupported: '20 and earlier'
 - platform: 'iOS'
   chipsets: 'Arm64'
-  supported: '17'
-  besteffort: '12 to 16, 18'
+  supported: '12 to 18'
+  ci-tested: '16, 17'
   unsupported: '11 and earlier'
 - platform: 'macOS'
   chipsets: 'x64, Arm64'
-  supported: '<abbr title="Ventura">13</abbr>, <abbr title="Sonoma">14</abbr>'
-  besteffort: '<abbr title="Mojave">10.14</abbr> to <abbr title="Monterey">12</abbr>, <abbr title="Sequoia">15</abbr>'
-  unsupported: '<abbr title="High Sierra">10.13</abbr> and earlier'
+  supported: 'Mojave (10.14) to Sequoia (15)'
+  ci-tested: 'Ventura (13), Sonoma (14)'
+  unsupported: 'High Sierra (10.13) and earlier'
 - platform: 'Windows'
   chipsets: 'x64, Arm64'
-  supported: '10'
-  besteffort: '11'
+  supported: '10, 11'
+  ci-tested: '10'
   unsupported: '8 and earlier'
 - platform: 'Debian (Linux)'
   chipsets: 'x64, Arm64'
-  supported: '11, 12'
-  besteffort: '10'
+  supported: '10, 11, 12'
+  ci-tested: '11, 12'
   unsupported: '9 and earlier'
 - platform: 'Ubuntu (Linux)'
   chipsets: 'x64, Arm64'
   supported: '20.04 LTS, 22.04 LTS'
-  besteffort: '24.04 LTS'
+  ci-tested: '22.04 LTS'
   unsupported: '23.10 and earlier non-LTS'
 - platform: 'Chrome (Web)'
   chipsets: 'N/A'
   supported: '[Latest 2](https://chromereleases.googleblog.com/search/label/Stable%20updates)'
-  besteffort: '96 to latest 2'
+  ci-tested:
```

Comment:
@yjbanov what browser versions do we have on CI?

Reply:
Engine:
  - Chrome: 119
  - Firefox: 106
  - Safari: depends on the version of macOS 13 (if latest, then it's Safari 18)
Framework:
  - Chrome: 125